pkg/configtx/policies.go

```diff
@@ -178,9 +166,9 @@ func (c *ConfigTx) RemoveConsortiumOrgPolicy(consortiumName, orgName, policyName
 
 }
 
-// AddOrdererPolicy modifies an existing orderer policy configuration.
-// When the policy exists it will overwrite the existing policy.
-func (c *ConfigTx) AddOrdererPolicy(modPolicy, policyName string, policy Policy) error {
+// SetOrdererPolicy sets the policy map for the existing orderer configuration Group.
```

Comment:
these policy set functions don't actually set the policy map, instead they effectively "add" a policy to that policy map (see line 172). It's this act of adding the policy that overwrites the existing value since the policy map is well in fact a map. Maybe something along the lines of `SetOrdererPolicy sets the specified policy in the orderer group's config policy map.` Same comment applies to all of the `SetXXXPolicy` functions in this PR.

Reply:
done